packages/util-crypto/src/hd/ethereum/index.ts

```diff
@@ -0,0 +1,224 @@
+// Copyright 2017-2021 @polkadot/util-crypto authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import * as crypto from 'crypto';
+// const Buffer = require('safe-buffer').Buffer;
+import { Buffer } from 'safe-buffer';
+
+// const assert = require('assert');
+import { assert } from '@polkadot/util';
+
+import { secp256k1KeypairFromSeed, secp256k1PrivateKeyTweakAdd } from '../..';
+
+const MASTER_SECRET = Buffer.from('Bitcoin seed', 'utf8');
+const HARDENED_OFFSET = 0x80000000;
+// Bitcoin hardcoded by default, can use package `coininfo` for others
+const BITCOIN_VERSIONS = { private: 0x0488ADE4, public: 0x0488B21E };
+
+interface Versions{
+  private:number;
+  public:number;
+}
+
+// function to retype Buffer to ArrayBuffer in a controlled manner
+function bufferAsArrayBuffer (buffer:Buffer):ArrayBuffer {
```

Comment:
```suggestion
function bufferAsArrayBuffer (buffer:Buffer): ArrayBuffer {
```

These are all nits. Really not sure why eslint doesn't force a space here. (really should between `:` and `'|` - which is used extensively here)

Reply:
Master now will auto-fix these when running with `yarn lint`.